crates/db_views/src/structs.rs

```diff
@@ -127,3 +129,19 @@ pub struct LocalUserDiscussionLanguageView {
   pub local_user: LocalUserSettings,
   pub language: Language,
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CustomEmojiView {
+  pub custom_emoji: CustomEmoji,
+  pub keywords: Option<Vec<CustomEmojiKeyword>>,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CustomEmojiViewForm {
```

Comment:
No views have forms, they are only for viewing data, not inserting it. 

It looks like you're missing the impl for these in `db_schema/impls/..`

Reply:
That's kinda where my question comes from. I wasn't sure where to put a composite struct of two database tables. I'll expand on this in comment in other issue around impl